### PR TITLE
Update container provision scripts

### PR DESCRIPTION
**Pull request by danrodrig**
- Add CentOS 8
- Remove Fedora 30
- Remove procps-ng package from Fedora provision scripts

### CODE REVIEW

File: containers/provision/centos:8.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	       make \
+	       rpm-build \
+	       createrepo_c \
+	       valgrind \
+	       git-daemon \
+	       python3-flask \
+	       python3-requests \
+	       python3-pytest \
+	       python3-six \
+	       procps-ng
```
Comment (StykMartin) [inline containers/provision/centos:8.sh:14]: Why do we need `procps-ng`? I guess it should be running without it

File: containers/provision/centos:8.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+dnf config-manager -y --enable PowerTools
```
Comment (StykMartin) [inline containers/provision/centos:8.sh:16]: `PowerTools` == `AppStream` ?

File: containers/provision/centos:8.sh (hunk 1 above)
Comment (danrodrig) [reply containers/provision/centos:8.sh:14]: Left over from the pidof/pgrep saga. I'll remove it from Fedora too.

File: containers/provision/centos:8.sh (hunk 2 above)
Comment (danrodrig) [reply containers/provision/centos:8.sh:16]: Ne, AppStream is already enable in the image.
```
Name         : libarchive-devel
Version      : 3.3.2
Release      : 8.el8_1
Architecture : x86_64
Size         : 131 k
Source       : libarchive-3.3.2-8.el8_1.src.rpm
Repository   : PowerTools
```
```
Name         : json-c-devel
Version      : 0.13.1
Release      : 0.2.el8
Architecture : x86_64
Size         : 44 k
Source       : json-c-0.13.1-0.2.el8.src.rpm
Repository   : PowerTools
```

Comment (StykMartin) [reply containers/provision/centos:8.sh:16]: CRB then I guess..

**Pull request merged**